conda/environments/all_cuda-122.yaml

```diff
@@ -3,16 +3,17 @@
 channels:
 - legate
 - legate/label/experimental
+- legate/label/rc
```

Comment:
Actually, one thing I could see is that we should order the rc label before the experimental one (not that it matters right now).